### PR TITLE
Release 2.22 - Adding Solana Support

### PR DESCRIPTION
**Pull request by RamRamez**

### CODE REVIEW

File: lang/es.json [hunk 1]
```diff
@@ -277,7 +279,7 @@
 	"label.done": "Listo",
 	"label.donor": "Donante",
 	"label.donors_to_higher_ranked_projects": "Los donantes de los proyectos con ranking más alto reciben más GIVbacks.",
-	"label.donors_to_verified": "Los donantes a proyectos verificados son recompensados con GIV. Boostea este proyecto para aumentar su porcentaje de recompensas y hacerlo más visible en la página de proyectos!",
+	"label.donors_to_verified": "Los donaciones en Ethereum a proyectos verificados son recompensados con GIV. Boostea este proyecto para aumentar su porcentaje de recompensas y hacerlo más visible en la página de proyectos!",
```
Comment (mateodaza) [inline lang/es.json:282]: Let's change it to

"Las donaciones en Ethereum a proyectos verificados son recompensadas con GIV. Boostea este proyecto para aumentar su porcentaje de recompensas y hacerlo más visible en la página de proyectos!"

File: lang/es.json [hunk 2]
```diff
@@ -229,6 +230,7 @@
 	"label.credit_card_info": "Información bancaria",
 	"label.crypto_made_easy": "Cripto de manera fácil",
 	"label.curate_projects": "Cura Proyectos",
+	"label.currently_this_feature_is_only_available": "Currently this feature is only available on EVM chains.",
```
Comment (mateodaza) [inline lang/es.json:233]: This in spanish would be

"Actualmente esta funcionalidad solo esta disponible en cadenas EVM"

**Review by mateodaza — CHANGES_REQUESTED** [review]
@RamRamez awesome man, left a couple of tiny changes on the spanish translations but besides that it LGTM

File: lang/es.json (hunk 1 above)
Comment (RamRamez) [reply lang/es.json:282]: Thanks Mateo. I fixed it here https://github.com/Giveth/giveth-dapps-v2/pull/3689

File: lang/es.json (hunk 2 above)
Comment (RamRamez) [reply lang/es.json:233]: also this one is fixed here https://github.com/Giveth/giveth-dapps-v2/pull/3689

File: lang/es.json (hunk 1 above)
Comment (aminlatifi) [reply lang/es.json:282]: @mateodaza next time try amazing "Add suggestion" feature in your review, you would like it
image
Then, the peer will push/apply your suggestion easily with a single button click.

**Review by MohammadPCh — APPROVED** [review]
LGTM, Thanks

**Review by alireza-sharifpour — APPROVED** [review]
Thanks @RamRamez

**Pull request merged**